src/ui/QuerySuggestPreview/QuerySuggestPreview.ts

```diff
@@ -53,45 +46,12 @@ export class QuerySuggestPreview extends Component implements IComponentBindings
       defaultValue: 3,
       min: 1,
       max: 6
-    }),
-    /**
-     * The width of the preview container (in pixels).
-     *
-     * If this option is `undefined` or lower than the remaning space left by the suggestion,
-     * the component takes all the space left by the query suggestions.
-     */
-    previewWidth: ComponentOptions.buildNumberOption(),
-    /**
-     *  The width of the suggestion container (in pixels).
-     *
-     *  If the value is set to `0`, the width will adjust to the longest displayed query suggestion.
-     *
-     * **Default:** `250`
-     * **Minimum:** `0`
-     */
-    suggestionWidth: ComponentOptions.buildNumberOption({ defaultValue: 250, min: 0 }),
-    /**
-     *  The text to display at the top of the preview, which is followed by "`<SUGGESTION>`", where `<SUGGESTION>` is the hovered query suggestion.
-     *
-     * **Default:** The localized string `Query result items for`
-     */
-    headerText: ComponentOptions.buildLocalizedStringOption({ defaultValue: 'QuerySuggestPreview' }),
-    /**
-     *  The amount of focus time (in milliseconds) required on a query suggestion before requesting a preview of its top results.
-     *
-     * **Default:** `200`
-     */
-    executeQueryDelay: ComponentOptions.buildNumberOption({ defaultValue: 200 })
```

Comment:
This is now determined by which option ?

Reply:
Woops, I don't know why I removed this one, I'm adding it back.